### ISSUE TITLE
Form is not responding after clicking on Button then close Form

### ISSUE THREAD

**Issue by Lydia-Shi** [dotnet/winforms#6794]
* .NET Core Version: 
.NET 7.0.100-preview.3.22153.22

* Have you experienced this same bug with .NET Framework?: 
No

**Problem description:**
Form is not responding after clicking on button then close Form.
![Button_NotRespond](https://user-images.githubusercontent.com/54019172/156727801-926a838f-772b-4845-96c4-089936e84618.gif)

**Expected behavior:**
Form and VS all work well.

**Minimal repro:**
1.	Create a Winforms .NET Core application
2.	Drag and drop a Button control into Form designer
3.	Build and run 
4.	Click on button, then try to close Form

**More info:**
1.	This is a regression issue, and cannot repro in 7.0.100-preview.3.22152.16
2.	This issue affects many controls: TextBox, Button, Label, PropertyGrid, DataGridView, and …

**Comment by willibrandon** [comment]
When your application becomes unresponsive, what happens if you click Stop Debugging (Shift + F5) in VS?

I might be seeing the same or a similar issue in my environment.

**Comment by Lydia-Shi** [comment]
@willibrandon  After stopping debugging in VS, the following window is displayed.
![image](https://user-images.githubusercontent.com/54019172/156959154-da7371bd-de53-4482-8086-fc81a8a28f47.png)

**Comment by willibrandon** [comment]
Then I'm fairly certain I'm running into the same issue as you.

![StopDebuggingInProgress](https://user-images.githubusercontent.com/5017479/156964332-2121e506-5e66-4370-9849-998ceb7509d0.png)

If I can be of any assistance troubleshooting this, please let me know.

**Comment by dreddy-work** [comment]
@tommcdon is investigating this issue. Will update the thread as soon as we have a solution.

**Comment by WPMGPRoSToTeMa** [comment]
Note that I already reported this bug in dotnet/runtime#65664.

**Comment by tommcdon** [comment]
@v-qiashi would you mind trying to disable Intellitrace in Visual Studio to see if it works around the issue?  I believe that this issue is actually a Visual Studio problem and is tracked on Devdiv internal bug #'s [1495188](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1495188) and [1495179](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1495179).

**Comment by RussKie** [comment]
Closing as duplicate of https://github.com/dotnet/runtime/issues/65664.

**Comment by Lydia-Shi** [comment]
> @Lydia-Shi would you mind trying to disable Intellitrace in Visual Studio to see if it works around the issue? I believe that this issue is actually a Visual Studio problem and is tracked on Devdiv internal bug #'s [1495188](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1495188) and [1495179](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1495179).

@tommcdon  Yes, it's a workaround, after disabling Intellitrace, this issue will not occur.

**Comment by Cassie-Li01** [comment]
Verified on .NET 7.0.100-preview.4.22201.8, it is fixed. Please see the git as below:
![fixed](https://user-images.githubusercontent.com/96758262/161372926-4bac954f-73a3-4fc2-853d-cbc2f6fbc864.gif)
